packages/mui-joy/src/CssBaseline/CssBaseline.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { GlobalStyles } from '@mui/system';
+import { Theme, DefaultColorScheme, ColorSystem } from '../styles/types';
+import { CssBaselineProps } from './CssBaselineProps';
+
+/**
+ * Kickstart an elegant, consistent, and simple baseline to build upon.
+ */
+function CssBaseline({ children, enableColorScheme = false }: CssBaselineProps) {
+  return (
+    <React.Fragment>
+      <GlobalStyles
+        styles={(theme: Theme) => {
+          const colorSchemeStyles: Record<string, any> = {};
+          if (enableColorScheme) {
+            // The CssBaseline is wrapped inside a CssVarsProvider
+            (
+              Object.entries(theme.colorSchemes) as Array<[DefaultColorScheme, ColorSystem]>
+            ).forEach(([key, scheme]) => {
+              colorSchemeStyles[theme.getColorSchemeSelector(key).replace(/\s*&/, '')] = {
+                colorScheme: scheme.palette?.mode,
+              };
+            });
+          }
+          return {
+            html: {
+              WebkitFontSmoothing: 'antialiased',
+              MozOsxFontSmoothing: 'grayscale',
+              // Change from `box-sizing: content-box` so that `width`
+              // is not affected by `padding` or `border`.
+              boxSizing: 'border-box',
+              // Fix font resize problem in iOS
+              WebkitTextSizeAdjust: '100%',
+            },
+            '*, *::before, *::after': {
+              boxSizing: 'inherit',
+            },
+            'strong, b': {
+              fontWeight: theme.vars.fontWeight.lg,
```

Comment:
**Off-topic**

I'm not sold on the terminology used by the font-weight's scale. I wonder if a number (100, 200, etc.) or a full name (bold, medium, etc.) that matches the CSS spec wouldn't be more intuitive 🤷‍♂️.

Reply:
I'm fine to go with `bold`.